### DISCUSSION TITLE
copy files dialog improvements

### DISCUSSION THREAD

**Discussion by mikekazakov** [mikekazakov/nimble-commander#238] [Ideas]
When doing a Move or similar and the little sheet comes down with the oval button Show and Advanced Options label, you need to make that a little less weird. I suggest a normal Show Details button and Hide Details button like in the standard Print dialog. Please give it a keyboard short cut.

**Comment by m-cerveny** [comment]
I actually like the fact that the dialog is modal, however, its too cryptic and missing path, filename, such that in the time of need, it's possible to copy to other locations that the one in the other pane.

Total Commander or Crax get it right.
<img width="406" alt="crax copy dialog" src="https://cloud.githubusercontent.com/assets/11475311/25045830/6efa2412-212e-11e7-9583-4476fdda6264.png">

Would that be possible?
